### DISCUSSION TITLE
Overcalculated contributions

### DISCUSSION THREAD

**Discussion by d3ttl4ff** [community/community#114399] [Repositories]
### Select Topic Area

Question

### Body

So initially I had around 430+ total contributions. And then I renamed my main branch in a few of my repos. After some time my contributions got doubled, it showed around 800+ contributions. Then I tried renaming my main branch to the previous name but it didn't work. Any solutions for this?

**Comment by devashish2024** [comment]
I'm not sure how this happened, however I do not think so that there could be a reverse for this, GitHub does not provides reverting to specific commit points unless from PRs.

And specifically , GitHub staff will not be going to help in this case. Sadly, you might try to make a new repo (which is not a good idea on losing your commits). 

However, feel free to open a support ticket as they might either try to fix it as a bug (in case it is), or help you too. If you have any more doubts, feel free to ask me!

**Reply by d3ttl4ff** [reply]
Hmm. Yea I rather not lose my repos rn. I'll try opening a ticket then. Thanks anyways!!

**Reply by devashish2024** [reply]
Alright. Mark it as answer if you think there is nothiing else to be done in this discussion to keep the community forums clean, if I helped you.

Also do not attempt to delete your branch without thinking multiple times.
